draft-irtf-cfrg-hpke.md

```diff
@@ -94,6 +94,23 @@ informative:
     target: https://secg.org/sec1-v2.pdf
     date: 2009
 
+  BHK09:
+    title: "Subtleties in the Definition of IND-CCA: When and How Should Challenge-Decryption be Disallowed?"
+    target: https://eprint.iacr.org/2009/418
+    date: 2009
+    author:
+      -
+        ins: Mihir Bellare
+        org: University of California San Diego
+      -
+        ins: Dennis Hofheinz
+        org: CWI Amsterdam
+      -
+        ins: Eike Kiltz
+        org: CWI Amsterdam
+
+  SigncryptionDZ10: DOI.10.1007/978-3-540-89411-7
```

Comment:
I'm dubious about referencing signcryption... as I understand it, this is far more nascent than the other foundational work we cite.

Reply:
It is right that it there is a lot of movement in the field of signcryption. That is part of the reason why notions adapted to HPKE are in the works. I'll inline the notions, then we see if this looks alright.